integration-tests/src/test/resources/features/action-kafka.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Kafka Action tests
+
+  Background:
+    Given authenticate against Manager
+
+  @kafkaaction
+  Scenario: Send message to Kafka topic using the Kafka action
+    And create a new Kafka topic "topic1"
```

Comment:
Given that we use the same kafka cluster, can this test produce errors if multiple jobs are running in parallel?
They both can create/delete the same topic concurrently.

Reply:
Yes, I understand. There is already some logic for it. It will append prefix with random hash to the name. https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/985/files#diff-f6186aa2d4db523463b2612cf5b1f828ec29566cf37aa7a90287e95c1c1a55e7R85